python/composio/tools/toolset.py

```diff
@@ -153,6 +155,7 @@ def __init_subclass__(
                 f"description_char_limit is not set on {cls.__name__}, using 1024 as default"
             )
         cls._description_char_limit = description_char_limit or 1024
+        cls._action_name_char_limit = action_name_char_limit
```

Comment:
Consider adding a warning message when `action_name_char_limit` is not set, similar to how we handle `description_char_limit`. This would maintain consistency in the codebase:

```python
if action_name_char_limit is None:
    warnings.warn(
        f"action_name_char_limit is not set on {cls.__name__}, using no limit as default"
    )
```